packages/loot-core/src/server/accounts/transactions.ts

```diff
@@ -96,7 +102,7 @@ export async function batchUpdateTransactions({
     if (updated) {
       await Promise.all(
         updated.map(async t => {
-          if (t.account) {
```

Comment:
So updating a transaction should have already removed the category?  Why was it not working?  or is this just to catch existing things?

Reply:
Only if the account was changed, so you could add a category to a transaction after it was moved or created in the off budget account

Reply:
Actually I have no clue why it was not working. My change here wasn't working but I also couldn't find an method of adding a category after creation.